ubr/utils.py

```diff
@@ -132,6 +133,52 @@ def pairwise(lst):
     iterator = iter(lst)
     return izip(iterator, iterator)
 
+def enumerated(lst):
+    return dict(zip(range(1, len(lst) + 1), lst))
+
+def isint(x):
```

Comment:
it seems you want a *positive* integer, so https://docs.python.org/2/library/stdtypes.html#str.isdigit works best (and substitutes this function altogether)